app/assets/stylesheets/govuk-component/govspeak/_example.scss

```diff
@@ -7,9 +7,16 @@
 
 .govuk-govspeak {
   .example {
-    border-left: 10px solid $panel-colour;
-    padding-left: 1.5em;
-    margin: 2.5em 0;
+    border-left: 1em solid $panel-colour;
+    padding: 1em 0 1em 1em;
```

Comment:
We could move to using multiples of `$gutter-half` (it’s essentially a difference of 15px vs a computed 16px).

It'd be good to better understand Mia's intentions. I'm aware Mia's out of the office today, so let's hold merging this till Monday, and discuss it then.